frontend/src/pages/learn/index.tsx

```diff
@@ -158,60 +177,88 @@ const Learn: FC = () => {
-                        onClick={handleNext}
-                        disabled={isNextDisabled}
-                      >
-                        {t('quiz.navigation.start')}
-                      </Button>
-                    )}
-                    {currentStepIndex != 0 && (
-                      <div>
-                        <MobileStepper
-                          variant="progress"
-                          steps={10}
-                          position="static"
-                          activeStep={currentStepIndex}
-                          backButton={undefined}
-                          nextButton={undefined}
-                          classes={{
-                            progress: 'w-full',
-                          }}
-                        />
-                        <p className="text-center">{currentStepIndex} of 9</p>
+                  {showConfirmation ? (
+                    <div className='w-full text-center flex flex-col h-full'>
+                      <ErrorOutlineIcon className="text-9xl m-10 font-bold text-red-dark mx-auto" />
+                      <p className='mb-10'>{t('quiz.confirmation.sure')}</p>
+                      <div className='mt-auto'>
                         <Button
-                          onClick={handlePrev}
-                          disabled={isPrevDisabled}
-                          className="w-1/2 px-4 py-2 font-display font-bold normal-case text-primary-700 hover:bg-white"
+                          onClick={handleConfirmationCancel}
+                          className="border border-gray-300 mx-4 hover:border-gray-300 w-1/3 px-4 py-2 font-display font-bold normal-case text-primary-700 hover:bg-white bg-white shadow-none hover:shadow-none"
+                          variant="outlined"
                         >
-                          {t('quiz.navigation.previous')}
+                          {t('quiz.confirmation.no')}
                         </Button>
-                        {currentStepIndex === 9 ? (
-                          <Button
-                            onClick={handleNext}
-                            disabled={isNextDisabled}
-                            className="ml-auto w-1/2 rounded bg-primary-700 px-4 py-2 font-display font-bold normal-case text-white hover:bg-primary-800"
-                          >
-                            {t('quiz.navigation.submit')}
-                          </Button>
-                        ) : (
+                        <Button
+                          onClick={handleCloseModal}
+                          className="w-1/3 rounded mx-4 bg-primary-700 px-4 py-2 font-display font-bold normal-case text-white hover:bg-primary-800"
+                        >
+                          {t('quiz.confirmation.yes')}
+                        </Button>
+                      </div>
+                    </div>
+                  ):(
+                    <>
+                      {renderComponent()}
```

Comment:
This looks strange. Why isn't it `<SomeComponent />`?

Reply:
https://formikwizard.manishjangir.com/docs/introduction/formik-wizard-provider/#example

Reply:
Okay.. I'll mark as resolved but it's still kinda weird to me.